src/Makefile

```
@@ -86,6 +86,13 @@ DISABLE_SYSC?=no
 DISABLE_CISCAT?=no
 DISABLE_STRIP_SYMBOLS?=no
 
+ifeq (${uname_S}, Linux)
+EXECUTABLE_FILES := $(shell find . -maxdepth 1 -type f ! -name "*.so"  -perm /a=x)
+EXECUTABLE_FILES += $(shell find . -type f -not -path "./external/*" -name "*.so" -perm /a=x)
+EXECUTABLE_FILES:= $(EXECUTABLE_FILES:./%=%)
+endif
```

Winagent is compiled also in Linux OS, so it is needed to also filter by target
```suggestion
ifeq (${uname_S}, Linux)
ifneq (${TARGET}, winagent)
EXECUTABLE_FILES := $(shell find . -maxdepth 1 -type f ! -name "*.so"  -perm /a=x)
EXECUTABLE_FILES += $(shell find . -type f -not -path "./external/*" -name "*.so" -perm /a=x)
EXECUTABLE_FILES:= $(EXECUTABLE_FILES:./%=%)
endif
endif
```

---

But wouldn't the executable files have to be looked up in a different way? i.e. by filtering .dlls and .exes?